api/src/project_view_history.ts

```diff
@@ -11,6 +11,7 @@ import { BusinessEvent } from "./service/domain/business_event";
 import { ServiceUser } from "./service/domain/organization/service_user";
 import * as Project from "./service/domain/workflow/project";
 import * as Subproject from "./service/domain/workflow/subproject";
+import logger from "./lib/logger";
```

Comment:
not sorted

Reply:
and below (can't comment there) the request body related stuff is dead code - the request can't have a body